bindings/python/tersets/__init__.py

```diff
@@ -98,10 +112,13 @@ def decompress(values: bytes) -> List[float]:
 
     decompressed_values = __UncompressedValues()
 
-    configuration = __Configuration(0, 0.0)
+    if (type(method) != Method):
+        # Method does not exists, raise error, and show available options
```

Comment:
Missing punctuation at the end of the comment.

Reply:
fixed